py/bitbox02/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -407,3 +419,22 @@ def get_info(self) -> Tuple[str, Platform, Union[BitBox02Edition, BitBoxBaseEdit
         unlocked_byte = response[0]
         unlocked = {0x00: False, 0x01: True}[unlocked_byte]
         return (version_str, platform, edition, unlocked)
+
+    def check_version(self) -> Tuple[bool, semver.VersionInfo]:
```

Comment:
returning the constant `MIN_BITBOX02_MULTI_FIRMWARE_VERSION` is not helpful imho, can just use `MIN_BITBOX02_MULTI_FIRMWARE_VERSION` wherever you need it.